qds_sdk/template.py

```diff
@@ -0,0 +1,137 @@
+"""
+The Template Module contains the base definition for Executing Templates
+"""
+import json
+import os
+
+from argparse import ArgumentParser
+from qds_sdk.qubole import Qubole
+from qds_sdk.resource import Resource
+
+class TemplateCmdLine:
+    
```

Comment:
Can you add comment for the class.  Helps in document generation using sphinx.  More important would be the class Template. 
Also comment on each important function(about the input parameters and output format if any) within Template class which you feel will be used by the sdk-user and needs to be exposed via the python code document.